test/async-hooks/test-httpparser.response.js

```diff
@@ -19,11 +19,8 @@ const hooks = initHooks();
 hooks.enable();
 
 const request = Buffer.from(
-  'HTTP/1.1 200 OK' + CRLF +
-  'Content-types: text/plain' + CRLF +
-  'Content-Length: 4' + CRLF +
-  CRLF +
-  'pong'
+  `HTTP/1.1 200 OK${CRLF}Content-types: text/plain${CRLF}` +
+  `Content-Length: 4${CRLF}${CRLF}pong`
```

Comment:
>I thought the capitalization was by design since this is a test file.

IDK, looks like the test just checks that you get a response. You think it's checking that a different capitalisation is still valid?